whitepaper/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM pandoc/latex:2.12
```

Comment:
Why do you use Docker in the whitepaper? It seems that latex container is not needed because whitepaper is markdown style.

Reply:
It's incidental: I need pandoc for the Markdown => HTML conversion. I picked their latex image instead of the one they called 'core' because according to their docs the 'core' is lacking most features (some may not be related to latex) compared to the image called 'latex'.

Reply:
@petermetz Thanks for your replying. I understand that you need pandoc.
However, this is just a document, so I think it is enough for users to install pandoc by themselves. I don't know why you want to offer it as a Docker container, but could you tell me why you need it?

Reply:
> so I think it is enough for users to install pandoc by themselves.

It's for the convenience of the users/contributors. My guiding principle here is that anything that takes more than 3 clicks to achieve is something I want to make simpler to reduce cognitive overhead especially for newcomers.

> I don't know why you want to offer it as a Docker container

I don't mean to use the docker container as a way of distributing the document itself.
This container is for the tool-chain used to build/compile different formats of the document such as HTML and PDF. 
There are other issues open with tasks to have automated publishing of the most up to date version of the whitepaper in PDF format as well and having a container will play nicely into that.

@takeutak

Reply:
@takeutak Kindly requesting that you review or follow-up on the review discussion within this pull request (whichever is applicable). Thank you very much in advance.